### ISSUE TITLE
Scroll position on safari

### ISSUE THREAD

**Issue by gizmotangle** [alvarotrigo/react-fullpage#418]
I am using fullpage version 4 with big content pages. On Safari Desktop the scroll position in sections does not start on top, it starts on scroll position (or somewhere) the user has scrolled before. How can I fix this?

**Comment by alvarotrigo** [comment]
> On Safari Desktop the scroll position in sections does not start on top, it starts on scroll position (or somewhere) the user has scrolled before. How can I fix this?

It sounds like you want to use the paid extension [ScrollOverflow Reset](https://alvarotrigo.com/fullPage/extensions/scrolloverflow-reset.html).

**Comment by gizmotangle** [comment]
I only want to have the same behave as I have in all other browsers like chrome/ff ... its all fine on Linux/Windows/Apple instead of Apple Safari. Why have I to purchase an extension for Safari only when it works on other browsers as expected??

**Comment by alvarotrigo** [comment]
I probably misunderstood your question. 
ScrollOverflow always stays on the position the  user left it when leaving that section. So not quite sure why you say this behaves in a different way in other browsers.

Can you provide an example or video of the issue you are having?
Are you able to reproduce your issue on this example page?
https://alvarotrigo.com/fullPage/examples/scrolling.html

**Comment by alvarotrigo** [comment]
Maybe you are talking about the kinetic/inertial scroll?

If that's the case there's an opened issue about this [on the main JS repository](https://github.com/alvarotrigo/fullPage.js/issues/4557). No solution has been found yet. Feel free to continue the conversation there.

**Comment by gizmotangle** [comment]
Yes I can reproduce it on your example page: https://alvarotrigo.com/fullPage/examples/scrolling.html

With FF/Chrome on my ubuntu scrolling down from first slide starts the content/scrolling position on the next slide on Top (as expected), scrolling on safari starts scrolling from "somewhere", but not on top of the next section.

**Comment by gizmotangle** [comment]
and yes... my problems seems to be very similar to  https://github.com/alvarotrigo/fullPage.js/issues/4557, if I scroll "fast" scrolling goes nearly to end of next slide, If I scroll very "sensitive" I reach the top of next slide

**Comment by alvarotrigo** [comment]
> and yes... my problems seems to be very similar to [alvarotrigo/fullPage.js#4557](https://github.com/alvarotrigo/fullPage.js/issues/4557), if I scroll "fast" scrolling goes nearly to end of next slide, If I scroll very "sensitive" I reach the top of next slide

Yeap, the same issue then.
Let's continue the conversation on [the original issue](https://github.com/alvarotrigo/fullPage.js/issues/4557). 

I'll close this topic to have it centralised in a single place.
